src/qt/askpassphrasedialog.cpp

```diff
@@ -192,7 +192,7 @@ void AskPassphraseDialog::accept()
                     QMessageBox::critical(this, tr("Wallet unlock failed"),
                                           tr("The passphrase entered for the wallet decryption is incorrect. "
                                              "It contains a null character (ie - a zero byte). "
-                                             "If the passphrase was set with a version of this software prior to 21.0, "
+                                             "If the passphrase was set with a version of this software prior to 23.0, "
```

Comment:
ec99c2f4fa289ff004af58618fba00c6088670ca: nit: merge it into original commit maybe